standalone-metastore/pom.xml

```diff
@@ -29,6 +29,7 @@
     <module>metastore-common</module>
     <module>metastore-server</module>
     <module>metastore-tools</module>
+    <module>metastore-catalog</module>
```

Comment:
`metastore-iceberg-catalog` or `metastore-iceberg-rest-catalog` is good to me. I don't want to confuse iceberg rest catlog with HMS catalog HIVE-18685.  cc @deniskuzZ

Reply:
https://github.com/apache/hive/pull/5606#discussion_r1961515595